session5/index.md

```diff
@@ -160,6 +104,38 @@ animals=["donkey",'giraffe','elephant',"chimpanzee"]
 
 If you want to find code that is both syntactically correct and using good style the best place to look is going to be the docs (mdn for JavaScript and ruby-doc for Ruby). The code examples are a great place to look to see if you are writing your code in a similar way. Also, keep in mind that things like casing of variables is another good way to stay consistent and maintain good style.
 
+## Error Messages
+
+Error messages are a beautiful thing. Learning how to read errors is very important in programming because they can tell you <em>exactly</em> where you made a mistake.
+
+But error messages can be overwhelming because they may overload you with information or use terms you may not understand.
+
+What are some strategies that you have already been using to interpret error messages?
+
+### Program Specific Breakout Groups (20 minutes)
+
+After watching a brief demo from the instructor, complete the following exercise with program-specific breakout groups:
+
+<div class="try-it">
+  <h3>Mythical Creatures: A Primer</h3>
+  <p>Everyone should do a fist to five to show their comfort with error messages. Whoever is the least comfortable with errors will share their screen. Whoever is sharing their screen will be the driver. Every other person in the room will be a navigator.</p>
+    <ol>
+      <li>Clone and cd into the repo:</li>
+      <ul>
+        <li><a href="https://github.com/corneliusellen/mythical-creatures-primer-fe">Frontend version</a></li>
+        <li><a href="https://github.com/corneliusellen/mythical-creatures-primer">Backend version</a></li>
+      </ul>
+      <li>Run the file with the command <code>ruby yeti.rb</code> for BE or <code>node yeti.rb</code> for FE.</li>
+      <li>Discuss with your group what you think the error message means.</li>
+      <li>Fix the error.</li>
+      <li>Repeat step 2-4 until there are no more errors. You will know you are making progress if the error messages change.</li>
+    </ol>
+
+  <p>You've corrected all the errors when you see the single line <code>We'll have to outwit the Bumble with our superior intelligence</code> outputted to your terminal.</p>
+
+  <p><b>NOTE: To fix the errors, do not delete any <em>lines</em> of code. You may only update/add code.</b></p>
+</div>
+
```

Comment:
I think a space for q & a or synthesis would be really helpful here. 

* How did it go?
* What are the main takeaways from this section? Could ask and maybe tell them what you want them to take away.